modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImpl.java

```diff
@@ -805,6 +805,10 @@ private void _extractIndexableAttributes(
 		for (DDMFormFieldValue ddmFormFieldValue : ddmFormFieldValues) {
 			DDMFormField ddmFormField = ddmFormFieldValue.getDDMFormField();
 
+			if (ddmFormField == null) {
+				return;
```

Comment:
@jvelalr please change to `continue;` instead

Reply:
@carolmariaabb done!, thanks.